docupload/views.py

```diff
@@ -19,11 +19,20 @@ def index(request):
     template = loader.get_template('docupload/index.html')
     form = DocUploadForm()
 
-    context = {
-        'doc_list': doc_list,
-        'form': form,
-    }
-    return HttpResponse(template.render(context, request))
+    context = {'doc_list': doc_list, 'form': form, }
+    return render(request, "docupload/index.html", context)
+
+
```

Comment:
Login required views . Can be added later. see this https://docs.djangoproject.com/en/1.9/topics/auth/default/#the-login-required-decorator